src/concrete/ml/onnx/onnx_utils.py

```diff
@@ -406,14 +407,20 @@
     "Less": numpy_less,
     "LessOrEqual": numpy_less_or_equal,
 }
+# All numpy operators used for tree-based models that support auto rounding
+ONNX_COMPARISON_OPS_TO_ROUNDED_NUMPY_IMPL_BOOL = {
```

Comment:
small detail but maybe `ONNX_COMPARISON_OPS_TO_ROUNDED_TREES_NUMPY_IMPL_BOOL` , as the most important part for these operators is that they are meant for trees only imo